citizen/citizen.c

```diff
@@ -9,129 +9,151 @@
 int counter = 0;
 
 
-// compare two Citizen objects based on the specified sort type.
-// Returns the result of the comparison.
-int cmpCtz(const Citizen a, const Citizen b, Type sort) {
+/*
+ * Compare two Citizen objects based on the specified sort type.
+ * Returns the result of the comparison.
+ */
+int compareCitizen(const Citizen a, const Citizen b, Type sort)
+{
     // Compare based on the sorting type provided
-    if (sort == FirstName) {
-        return strcmp(a.firstName, b.firstName);
+    if (sort == first_name)
+    {
+        return strcmp(a.first_name, b.first_name);
     }
 
-    if (sort == MiddleName) {
-        return strcmp(a.middleName, b.middleName);
+    if (sort == middle_name)
+    {
+        return strcmp(a.middle_name, b.middle_name);
     }
 
-    if (sort == LastName) {
-        return strcmp(a.lastName, b.lastName);
+    if (sort == last_name)
+    {
+        return strcmp(a.last_name, b.last_name);
     }
 
-    if (sort == CitizenId) {
-        return a.citizenId - b.citizenId;
+    if (sort == citizen_id)
+    {
+        return a.citizen_id - b.citizen_id;
     }
-    if (sort == BirthDate) {
-        return strcmp(a.birthDate, b.birthDate);
+
+    if (sort == birth_date)
+    {
+        return strcmp(a.birth_date, b.birth_date);
     }
     return 0;
 }
 
 
 // check if a Citizen's attribute contains a keyword.
-int com(const Citizen a, const char *keyword, Type type) {
+int com(const Citizen a, const char *keyword, Type type) 
```

Comment:
make the variable name of the citizen to be descriptive

Reply:
Renamed that variable to citizen_a. Thanks.